p2p/module_test.go

```diff
@@ -127,3 +148,18 @@ func Test_Create_configureBootstrapNodes(t *testing.T) {
 		})
 	}
 }
+
+// TECHDEBT(#609): move & de-duplicate
```

Comment:
I will start doing this as well. Definitely consider sharing `// TECHDEBT(#609): move & de-duplicate` with the team as a suggestion